scripts/agents/includes/sync-issue-fields.cjs

```
@@ -0,0 +1,251 @@
+  const typeMapping = config?.project_field_mappings?.Type ?? {};
+  const enabledTypes = new Set(
+    (config?.organization_issue_fields?.enabled_issue_types ?? []).map((t) =>
+      t.toLowerCase(),
+    ),
+  );
+
+  // Derive target native type name from labels
+  const targetTypeName = deriveTypeFromLabels(labelNames, typeMapping);
+
+  if (!targetTypeName) {
+    console.info(
```

ESLint

[error] 'console' is not defined.

(no-undef)